dwl.desktop

```diff
@@ -0,0 +1,5 @@
+[Desktop Entry]
+Name=DWL
+Comment=dwm for Wayland
+Exec=/usr/local/bin/dwl
```

Comment:
Just use `dwl`

Reply:
Err... I meant just replace `/usr/local/bin/dwl` with `dwl`, sorry for the ambiguity